x/modelinfo/genesis.go

```diff
@@ -53,24 +53,25 @@ func ValidateGenesis(data GenesisState) error {
 			return sdk.ErrUnknownRequest(fmt.Sprintf("Invalid ModelInfo: Missed SKU. Value: %v", record))
 		}
 
-		if record.HardwareVersion == "" {
+		if record.HardwareVersion == 0 {
 			return sdk.ErrUnknownRequest(fmt.Sprintf("Invalid ModelInfo: Missed HardwareVersion. Value: %v", record))
 		}
+		// TODO we should reuse validation
```

Comment:
I believe we should have validation for other fields here (not only HardwareVersion), shouldn't we?